# HG changeset patch
# User Rik <octave@nomad.inbox5.com>
# Date 1326245738 28800
#      Tue Jan 10 17:35:38 2012 -0800
# Branch stable
# Node ID c27491205e48fbb58151ecf672d8d04fc1147f57
# Parent  053404893d027370ebb22f02db04c7fd83d3afc7
oct-parse.yy: Remove unused variable to stop compiler warning.

* oct-parse.yy: Remove unused variable to stop compiler warning.

diff --git a/src/oct-parse.yy b/src/oct-parse.yy
--- a/src/oct-parse.yy
+++ b/src/oct-parse.yy
@@ -1894,18 +1894,16 @@ fold (tree_binary_expression *e)
   frame.protect_var (discard_warning_messages);
 
   discard_error_messages = true;
   discard_warning_messages = true;
 
   tree_expression *op1 = e->lhs ();
   tree_expression *op2 = e->rhs ();
 
-  octave_value::binary_op op_type = e->op_type ();
-
   if (op1->is_constant () && op2->is_constant ())
     {
       octave_value tmp = e->rvalue1 ();
 
       if (! (error_state || warning_state))
         {
           tree_constant *tc_retval
             = new tree_constant (tmp, op1->line (), op1->column ());
